pkg/controller/operators/olm/operator.go

```diff
@@ -1097,7 +1097,7 @@ func (a *Operator) syncClusterServiceVersion(obj interface{}) (syncError error)
 	}
 
 	// status changed, update CSV
-	if !(outCSV.Status.LastUpdateTime == clusterServiceVersion.Status.LastUpdateTime &&
+	if !(outCSV.Status.LastUpdateTime.Equal(clusterServiceVersion.Status.LastUpdateTime) &&
```

Comment:
Testing this would be interesting

I suppose we could use the downward API to get to the api-server and ask it questions about the resource utilization of the pod? Not sure if that even works. 

Although, since OLM bundles support prometheus objects, we could install an operator with a `ServiceMonitor` from a catalog which includes the prometheus operator (for upstream tests) and then spend the test inspecting the metrics ensuring CPU usage doesn't spike. This would work across upstream and openshift. It's a bigger effort but maybe worth it if we do need some kind of test. 

Maybe there's an easier way I'm not seeing